### PR TITLE
Dac6 340

### PR DESCRIPTION
**Pull request by ers2015112**

### CODE REVIEW

File: bt test [hunk 1]
```diff
@@ -0,0 +1,774 @@
+[33mcommit 912a71d232ca3755dfc99bf32abcfe12c0cec1b1[m[33m ([m[1;36mHEAD -> [m[1;32mDAC6-340[m[33m)[m
```
Comment (soljeggers) [inline bt test:1]: what is this file?

**Review by soljeggers — APPROVED** [review]
LGTM

File: test/controllers/WhatAreTheTaxNumbersForNonUKIndividualControllerSpec.scala [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+    "must redirect to the next page when valid data is submitted" in {
+
+      val mockSessionRepository = mock[SessionRepository]
+
+      when(mockSessionRepository.set(any())) thenReturn Future.successful(true)
+
+      val application =
+        applicationBuilder(userAnswers = Some(emptyUserAnswers))
+          .overrides(
+            bind[Navigator].toInstance(new FakeNavigator(onwardRoute)),
```
Comment (iservport) [inline test/controllers/WhatAreTheTaxNumbersForNonUKIndividualControllerSpec.scala:138]: Do we have a NavigatorSpec for this?

**Review by iservport — COMMENTED** [review]
LGTM

File: app/utils/CheckYourAnswersHelper.scala [hunk 3]
```diff
@@ -43,27 +43,27 @@ class CheckYourAnswersHelper(userAnswers: UserAnswers)(implicit messages: Messag
...
-        actions = List(
-          Action(
-            content            = msg"site.edit",
-            href               = routes.WhatAreTheTaxNumbersForUKIndividualController.onPageLoad(CheckMode).url,
-            visuallyHiddenText = Some(msg"site.edit.hidden".withArgs(msg"whatAreTheTaxNumbersForUKIndividual.checkYourAnswersLabel"))
-          )
-        )
-      )
-  }
+//  def whatAreTheTaxNumbersForUKIndividual: Option[Row] = userAnswers.get(WhatAreTheTaxNumbersForUKIndividualPage) map {
```
[the remark this answers is not in the document]
Comment (iservport) [reply app/utils/CheckYourAnswersHelper.scala:53]: Ok, I see line 1050.

File: test/controllers/WhatAreTheTaxNumbersForNonUKIndividualControllerSpec.scala (hunk 2 above)
Comment (iservport) [reply test/controllers/WhatAreTheTaxNumbersForNonUKIndividualControllerSpec.scala:138]: Ok, I see line 1050.

**Pull request merged**